app/src/main/java/com/brandon/forzenbook/network/LoginCodeResponse.kt

```diff
@@ -0,0 +1,6 @@
+package com.brandon.forzenbook.network
+
+data class LoginCodeResponse(
```

Comment:
You can make Exceptions from the Repo how I suggested Hamdan do

Reply:
I am now catching this in the UseCase instead and returning a boolean from the repo. Let me know if this is the intended design, I might have misinterpreted it. I can create a custom exception if that's needed to catch this case. To me, this looks like the cleanest way to handle it, since I am only looking to really catch this one case, but this might not be the intended way.